server/data/mappers/timelineMapper.ts

```diff
@@ -15,7 +15,10 @@ const toTimelineEvent = (timelineEventResponse: TimelineEventResponse): Timeline
     reference: timelineEventResponse.reference,
     sourceReference: timelineEventResponse.sourceReference,
     eventType: timelineEventResponse.eventType,
-    prisonName: timelineEventResponse.prisonId, // (JIRA: 480) TODO: Lookup Prison name from the service
+    prison: {
+      prisonId: timelineEventResponse.prisonId,
+      prisonName: undefined, // This mapper cannot look up / resolve the prisonName; that has to be post-processed
```

Comment:
All the change in this PR is essentially related to this line!
We can't lookup the prison name within the mapper. ie. we have service classes that use/depend on mappers - we don't want a mapper that uses/depends on a service - it the wrong way round.
So, the mapper will leave the `prison.prisonName` field undefined, and the `TimelineService` will post-process it to update this field (not this PR)